src/UI/class-settings-page.php

```diff
@@ -401,7 +401,7 @@ private function initialize_basic_section(): void {
 		);
 
 		// Track logged-in users.
-		$h = __( 'By default, the plugin will track the activity of users that are logged into this site. You can change this setting to only track the activity of anonymous visitors. Note: You will no longer see the Parse.ly tracking code on your site if you browse while logged in.', 'wp-parsely' );
+		$h = __( 'Choose whether logged-in visitors should be tracked. You will no longer see the Parse.ly tracking code on your site if you browse while logged-in. Note: For WordPress multisite, a user must be logged-in to the current site to be considered logged-in.', 'wp-parsely' );
```

Comment:
Can we make the bit about WP multisite conditionally shown if this is actually a multisite install (`is_multisite()`)? No point in telling single-site install users something that doesn't affect them.

Reply:
Done!